SteamReview/SteamReview/Steam.cpp

```diff
@@ -226,19 +246,31 @@ bool FSteam::AddNewGameToCategory(const FGame InGame)
 
 			int CategoryChosen;
 			std::cin >> CategoryChosen;
-			CategoryChosen = ValidateInput(CategoryChosen, 1, CategoryContainer.GetCurrentCategoryAmount()) - 1;
+			ValidateInput(CategoryChosen, 1, CategoryContainer.GetCurrentCategoryAmount());
+			
+			//Subtract 1 to select the current object inside of the array
+			CategoryChosen = CategoryChosen - 1;
 
 			//adding the game to category
-			if (CategoryContainer.AddGameToCategory(CategoryChosen, InGame))
+			FCategory Category;
+			if(CategoryContainer.GetCategory(CategoryChosen, Category))
 			{
-				std::cout << "------------------------------------------\n" << std::endl;
-				std::cout << "The game is going to add to the category: " << CategoryContainer.GetCategory(CategoryChosen - 1).GetCategoryName() << std::endl;
-				return true;
+				if (CategoryContainer.AddGameToCategory(CategoryChosen, InGame))
+				{
+					std::cout << "------------------------------------------\n" << std::endl;
+					std::cout << "The game is going to add to the category: " << Category.GetCategoryName() << std::endl;
+					return true;
+				}
+				else
+				{
+					std::cout << "The game couldn't be added to the category: " << Category.GetCategoryName() << std::endl;
+					return false;
+				}
 			}
 			else
 			{
 				std::cout << "------------------------------------------\n" << std::endl;
-				std::cout << "The game couldn't be added to the category: " << CategoryContainer.GetCategory(CategoryChosen - 1).GetCategoryName() << std::endl;
+				std::cout << "Something is wrong" << std::endl;
 				return false;
 			}
 		}
```

Comment:
I see here that you actually repeat the behaviour here, so all the comments above actually still apply. Remember it's empty, not empy, and you made another mistake in the comment below, the user hasn't*